.github/workflows/multibuild.yaml

```diff
@@ -230,7 +230,7 @@ jobs:
           node-version: '20.17.0'
       - name: setup required npm version
         run: |
-          npm install -g npm@10.8.2
+          npm ci npm@10.8.2
```

Comment:
I don't think this change will persuade the scanner, as it's still not pinned to a hash

Do we actually need `npm@10.8.2` stated on this line, or can it just be picked up from the lock file?

Reply:
However - actually this line is irrelevant. Will remove